### PR TITLE
Update typo of func comments.

### PR DESCRIPTION
**Pull request by k82cn**
typo of func comments.

### CODE REVIEW

**Comment by k82cn** [comment]
@k8s-bot bazel test this

**Comment by k82cn** [comment]
@k8s-bot unit test this

**Comment by k82cn** [comment]
@k8s-bot cvm gke e2e test this

**Comment by k82cn** [comment]
@k8s-bot unit test this

**Comment by cblecker** [comment]
LGTM

---

Reviewed 8 of 8 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/40642#-:-KbaTRYdlzwVckuB5WMz:bb6ywmx)*

File: plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go [hunk 1]
```diff
@@ -60,7 +60,7 @@ func getTolerationListFromPod(pod *v1.Pod) ([]v1.Toleration, error) {
 	return getAllTolerationPreferNoSchedule(tolerations), nil
 }
 
-// ComputeTaintTolerationPriority prepares the priority list for all the nodes based on the number of intolerable taints on the node
```
Comment (resouer) [inline plugin/pkg/scheduler/algorithm/priorities/taint_toleration.go]: LGTM, @davidopp Could you please add lgtm and approve label whenever you have time.

**Comment by k82cn** [comment]
will be handled in #42087 ; and I'll review it :).

**Pull request closed without merging**